rpm/tarantool.spec

```diff
@@ -35,14 +35,7 @@ BuildRequires: gcc-c++ >= 4.5
 BuildRequires: coreutils
```

Comment:
It seems, we should also revert 33830978870bbe8cdc3c23fa06ba0f141a960124.

Reply:
Looks okay. Done. Thanks!